client/purplecaffeine/trial.py

```diff
@@ -0,0 +1,127 @@
+"""Trial."""
+import os
+from typing import Optional, Union, List, Any
+import numpy as np
+from qiskit.providers import Backend
+from qiskit.circuit import QuantumCircuit
+from qiskit.quantum_info.operators import Operator
+
+from purplecaffeine.helpers import Configuration
+from purplecaffeine.backend import BaseBackend, LocalBackend
+
+
+class Trial:
+    """Trial class."""
+
+    def __init__(self, name: str, backend: Optional[BaseBackend] = None):
+        """Trial class for tracking experiments data.
+
+        Args:
+            name: name of trial
+            backend: backend to store data of trial. Default: local storage.
+            metrics: list of metric, like number of qubits
```

Comment:
Those are not args, they are properties :)

Reply:
Oh yes you right, only name and backend are args